### PR TITLE
Add toast notifications in Input, Edit and Delete pages.

### PR DESCRIPTION
**Pull request by priolette**

### CODE REVIEW

**Comment by Tauras-Rim** [comment]
Good job, the notifications look smooth

File: Energy Saver/Pages/Edit.cshtml.cs [hunk 1]
```diff
@@ -58,11 +65,13 @@ public async Task<IActionResult> OnPostAsync()
...
             {
                 await _context.SaveChangesAsync();
+                OnTaxEditSuccess();
             }
             catch (DbUpdateConcurrencyException)
             {
                 if (!TaxesExists(Taxes.ID))
                 {
+                    OnTaxEditError();
                     return NotFound();
```
Comment (Leminode) [inline Energy Saver/Pages/Edit.cshtml.cs:75]:
```suggestion
                    return RedirectToPage("./Index");
```

File: Energy Saver/Pages/Delete.cshtml.cs [hunk 2]
```diff
@@ -60,10 +68,36 @@ public async Task<IActionResult> OnPostAsync(int? id)
...
                 _context.Taxes.Remove(Taxes);
-                await _context.SaveChangesAsync();
+                try
+                {
+                    await _context.SaveChangesAsync();
+                    OnTaxDeleteSuccess();
+                }
+                catch (DbUpdateConcurrencyException)
+                {
+                    OnTaxDeleteError();
```
Comment (Leminode) [inline Energy Saver/Pages/Delete.cshtml.cs:78]:
```suggestion
                    OnTaxDeleteError();
                    return RedirectToPage("./Index");
```

File: Energy Saver/Pages/Input.cshtml.cs [hunk 3]
```diff
@@ -44,9 +51,35 @@ public async Task<IActionResult> OnPostAsync()
...
             _context.Taxes.Add(Taxes);
-            await _context.SaveChangesAsync();
+            try
+            {
+                await _context.SaveChangesAsync();
+                OnTaxInputSuccess();
+            } 
+            catch(DbUpdateConcurrencyException)
+            {
+                OnTaxInputError();
```
Comment (Leminode) [inline Energy Saver/Pages/Input.cshtml.cs:61]:
```suggestion
                OnTaxInputError();
                return RedirectToPage("./Index");
```

File: Energy Saver/Pages/Edit.cshtml.cs (hunk 1 above)
Comment (Leminode) [inline Energy Saver/Pages/Edit.cshtml.cs]:
```suggestion
                    return RedirectToPage("./Index");
```

Comment (priolette) [reply Energy Saver/Pages/Edit.cshtml.cs]: There are multiple NotFound() calls, I will change to RedirectToPage()

**Review by Leminode — APPROVED** [review]

**Pull request merged**